### ISSUE TITLE
Icons on dark mode

### ISSUE THREAD

**Issue by AlwaysLearningTech** [MicrosoftDocs/memdocs#1418]
Transparent backgrounds in images hide black lines while using dark mode. Updating images to white backgrounds would give greater accessibility. 

![image](https://user-images.githubusercontent.com/37163053/111547337-3d380f80-8736-11eb-8725-e58a32afd0c5.png)

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 95d2a103-e681-fa04-d3ff-3bffd1e41ca0
* Version Independent ID: 865c635a-2ee7-9c87-7a32-61464e5255ff
* Content: [Add Win32 app supersedence - Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/apps/apps-win32-supersedence)
* Content Source: [memdocs/intune/apps/apps-win32-supersedence.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/apps/apps-win32-supersedence.md)
* Service: **microsoft-intune**
* Sub-service: **apps**
* Technology: ****
* GitHub Login: @Erikre
* Microsoft Alias: **erikre**

**Comment by Erikre** [comment]
@AlwaysLearningTech - Thanks for the feedback. I have created an internal work item to track this fix. I'm closing this issue. Thanks again for point this out. - Erik
